relayer/chain/chain.go

```diff
@@ -7,8 +7,9 @@ type Message interface{}
 
 // Message from ethereum
 type EthereumOutboundMessage struct {
-	Call string
-	Args []interface{}
+	Call  string
+	Args  []interface{}
+	Nonce uint64
```

Comment:
I added the `Nonce` field here so that I can match it up with the stored nonce in the `incentivized` and `basic` inbound channels. This struct is used by the Ethereum layer too, but shouldn't have any effect since this new field isn't used in that relayer.